### PR TITLE
Update maintainers list

### PR DESCRIPTION
**Pull request by mattam82**
@andres-erbsen and @huynhtrankhanh if you would like to associate a picture you're welcome.
@blaisorblade can you check your details?

### CODE REVIEW

**Comment by huynhtrankhanh** [comment]
[You can take this one](https://user-images.githubusercontent.com/61727983/217887536-710507b0-2c41-4d89-b699-9e4c3de1986a.jpg)

File: pages/coq-team.html [hunk 1]
```diff
@@ -317,13 +323,27 @@
     <div class="position">Inria Advanced Researcher in Saclay, France</div>
     <div class="components">SSReflect, Mathematical Components, Notations</div>
   </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/blaisorblade.jpg" alt="Paolo Giuraso" width="200"/></div>
+    <div class="name"><a href="https://github.com/Blaisorblade">Paolo Giuraso</a></div><div class="nickname">blaisorblade</div>
```
Comment (Blaisorblade) [inline pages/coq-team.html:329]:
```suggestion
    <div class="name"><a href="https://github.com/Blaisorblade">Paolo G. Giarrusso</a></div><div class="nickname">blaisorblade</div>
```

**Comment by Blaisorblade** [comment]
For me, acceptable pictures are Avatar or none, but not the one you found despite the higher resolution — personal reasons.

File: pages/coq-team.html [hunk 2]
```diff
@@ -317,13 +323,27 @@
     <div class="position">Inria Advanced Researcher in Saclay, France</div>
     <div class="components">SSReflect, Mathematical Components, Notations</div>
   </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/blaisorblade.jpg" alt="Paolo Giuraso" width="200"/></div>
```
Comment (Blaisorblade) [inline pages/coq-team.html:328]:
```suggestion
      <img src="/files/team/blaisorblade.jpg" alt="Paolo G. Giarrusso" width="200"/></div>
```

File: pages/coq-team.html (hunk 1 above)
Comment (mattam82) [reply pages/coq-team.html:329]: Sorry for butchering your name there!

**Pull request merged**